ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -200,7 +201,9 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
 
     final var block =
         new Block(
-            newBlockHeader, new BlockBody(transactions, Collections.emptyList(), maybeWithdrawals));
+            newBlockHeader,
+            new BlockBody(
+                transactions, Collections.emptyList(), maybeWithdrawals, Optional.empty()));
```

Comment:
I was expecting deposits from newPayload's ExecutionPayloadV6110 to be wired in here, is there a reason it's not?

Reply:
Json-rpc is not really in the scope of this PR and I am trying to keep the changes in `api` as minimal as possible. Will wire it in the following PR